### DISCUSSION TITLE
你們覺得我們應該使用 Penpot 還是 Figma 作為設計協作工具比較方便？

### DISCUSSION THREAD

**Discussion by Tofuswang** [3dw/freemath#76] [Ideas]
我是參與這一次設計改善的設計師 Tofus

如題，我很好奇工程師對於開源設計工具 Penpot 的看法，比方說「將設計轉錄成程式碼的過程」是否有比Figma還要方便？

主要是考慮到若其他的設計師如果想要加入這個專案進行貢獻的話，可能並不是那麼熟悉如同 Penpot 這樣的開源工具。以及我在做設計的時候發現 Penpot 這個工具對於中文的支援度並沒有 figma 來得強（例如在輸入中文文字的時候整個App就會崩潰）。但又會擔心不是所有人都有Figma的專業版，可能沒有辦法調用輸出完整程式碼的功能，所以才想要上來詢問在這個專案中使用什麼樣的設計工具會比較適合大家協作，感謝！🙏

豆腐

**Comment by bestian** [comment]
@Tofuswang 
個人認為：團隊協件功能+免費這兩個優點下，Penpot會比較優先。
中文支援度的問題，可能可以：

1. 先用英文和英文假文當填充物，只要看得懂排版就好
2. 具體的錯誤回報，可以在出現時截圖，提到Penpot的議題區，由他們團隊慢慢去修好[https://github.com/penpot/penpot/issues](https://github.com/penpot/penpot/issues)

**Reply by Tofuswang** [reply]
已經和他們來回過信件，他們表示這個問題會在 9 月的更新時解決（也有其他 g0v 社群的貢獻者遇到這個問題，保險起見他等等會在他們的專案上開一張票確保這件事情會被解決，我也會在那張票底下更新我先前傳給他們的螢幕錄影）

不好意思讓你們等了這麼久🙏

**Reply by bestian** [reply]
@Tofuswang 謝謝聯絡Penpot團隊並確認9月更新時可解決中文化問題。
不必抱歉，大家本來都是在有餘裕的狀態下做開源專案。歡喜就好。

**Comment by leechiuhui** [comment]
@Tofuswang @bestian 
剛剛在[這篇文](https://yujih.substack.com/p/riveduolingo?utm_source=substack&publication_id=1078938&post_id=148464549&utm_medium=email&utm_content=share&utm_campaign=email-share&triggerShare=true&isFreemail=true&r=3imjca&triedRedirect=true)發現一款動畫「協作編輯器」與「跨平台播放器」， [Rive](https://rive.app/)。

想說會不會有Designers有興趣一起玩和協作一些數學動畫的表達？

另外,查了一下 Figma 和 [Penpot](https://penpot.app/pricing) 和這個 Rive 一樣都是有 pricing 升級方案的..

**Reply by bestian** [reply]
Rive看起來很強效, @Tofuswang 你覺得如何?
我對動畫設計非常不熟，這是 @leechiuhui 的長項. 我可以提供數學概念和跟著寫一點code和測試...

**Reply by leechiuhui** [reply]
哈哈..更正一下：動畫恰恰也是 @leechiuhui 我的極弱項，只是有很多會做Game Development 遊戲開發 2次元 3次元 人物、動作 場景設計的同事...而遊戲人物、動作和動畫其實有點不太同......
是說如果@bestian老師 可以提供數學概念和跟著寫一點code和測試... 和 @Tofuswang  g0v slack #designers 上有其他designers https://app.slack.com/client/T02G2SXKM/search 想一起玩 **開源** “數學相關” 動畫協作的話，好像可以試試看.... @Tofuswang 你覺得呢？

**Reply by bestian** [reply]
> 哈哈..更正一下：動畫恰恰也是 @leechiuhui 我的極弱項，只是有很多會做Game Development 遊戲開發 2次元 3次元 人物、動作 場景設計的同事...而遊戲人物、動作和動畫其實有點不太同...... 是說如果@bestian老師 可以提供數學概念和跟著寫一點code和測試... 和 @Tofuswang g0v slack #designers 上有其他designers https://app.slack.com/client/T02G2SXKM/search 想一起玩 **開源** “數學相關” 動畫協作的話，好像可以試試看.... @Tofuswang 你覺得呢？

@leechiuhui 邀請信已發~

**Comment by Tofuswang** [comment]
我有聽過 rive!
但之前都因為不知道要用它玩什麼所以就不知道該如何學，我覺得這個想法很有趣！

**Reply by bestian** [reply]
@Tofuswang 我開好了rive, workspace, 需要你的email來發邀請信, 可用email寄到我的信箱bestian@gmail.com, 謝謝~

**Comment by bestian** [comment]
@Tofuswang @leechiuhui 既然兩位都有興趣，我就先拋磚引玉了：

我把第一個動畫設計的草稿先用文字說明寫在

https://github.com/3dw/freemath/wiki/%E5%8B%95%E7%95%AB%E8%A8%AD%E8%A8%88%E8%8D%89%E7%A8%BF1:-%E4%BB%A5%E9%9D%A2%E7%A9%8D%E7%9A%84%E8%A7%92%E5%BA%A6%E7%9C%8B%E4%B9%98%E6%B3%95

如果覺得不夠清楚，我可以再補充手繪的分鏡稿~

**Reply by leechiuhui** [reply]
@bestian 老師，這個[面積與乘法.riv](https://github.com/3dw/freemath/commit/cd0af8fbc20ac7fc3b6c3652f0b745d08a1b5ca9)可否直接share在Rive的Freemath project裡？因為從github下載下來沒辦法在Rive web/desktop開啟...

<img width="1722" alt="截圖 2024-09-05 下午5 03 18" src="https://github.com/user-attachments/assets/6e7d3300-3332-4f4c-8ab7-7411aab50fdb">

**Reply by bestian** [reply]
@leechiuhui 剛才成功移動到共享區了~

(我第一次用Rive，不知道workspace 裡面還有分個人區跟共享區)

<img width="1354" alt="Rive" src="https://github.com/user-attachments/assets/16e4a002-06df-495d-813b-6b6d2639c27a">

**Comment by leechiuhui** [comment]
Thank you, teacher. I can now see the '面積與乘法.riv' file on the Rive web version, but I have to restart the Rive desktop version for it to be visible.
